AxonDeepSeg/ads_utils.py

```diff
@@ -0,0 +1,91 @@
+import os
+import sys
+import raven
+
+
+def init_ads():
+    """ Initialize ads for typical terminal usage
+    :return:
+    """
+    init_error_client()
+
+
+def get_dsn_filepath():
```

Comment:
I'm not sure of the value of this filepath indirection (there's a download in the SCT install, but I'm in the process of getting rid of it) compared to just hard-coding it in the code, with an environment variable to override.

Reply:
I guess I did it this way because I'm not comfortable with the best practice on setting up environment variables in shell installer scripts (or just in general) for multi-platform compatibility (we'd like to try and make ADS Windows compatible soon, see #91). Also, ADS only has a single command-line script, so I'd like tracking to work even for non-command line usage in our case. We also because we don't have an `sct_launcher`-type script that's called prior to the .py files being executed like you do in your SCT command-line bash scripts, which is what I think sets up your ENV variables right?